person.rb

```diff
@@ -0,0 +1,23 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..100_000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    of_age?
+  end
+
+  private
+
+  def of_age?
+    return true if @age >= 18
+
+    false
+  end
```

Comment:
- [ ] [OPTIONAL] You can implement this function as follows:
 ```ruby
  def is_of_age?
    @age >= 18
  end
```
since it's much easier to read and will make your code cleaner.